packages/builder/src/components/automation/AutomationBuilder/FlowChart/TestDataModal.svelte

```diff
@@ -81,7 +81,7 @@
   // Check the schema to see if required fields have been entered
   $: isError =
     !isTriggerValid(trigger) ||
-    !trigger.schema.outputs.required?.every(
+    !(trigger.schema.outputs.required || []).every(
```

Comment:
Why is this change needed?

Reply:
If we don't have any required field this will always be true, as `!undefined`. So, not having required fields, will always result on `isError = true`

Reply:
Ah makes sense!